src/ppl/src/IOPlacer.cpp

```diff
@@ -1507,11 +1507,12 @@ void IOPlacer::addVerLayer(odb::dbTechLayer* layer)
   ver_layers_.insert(layer->getRoutingLevel());
 }
 
-void IOPlacer::getPinsFromDirectionConstraint(Constraint& constraint)
+void IOPlacer::getPinsFromDirectionConstraint(Constraint& constraint,
+                                              int constraint_idx)
```

Comment:
How is the new argument used?

Reply:
Fixed this and the above/ I've started with these two changes, but they're not necessary anymore.